.github/workflows/CI-python.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       matrix:
         packageDirectory: ["interpret_community"]
         operatingSystem: [ubuntu-latest]
-        pythonVersion: [3.6, 3.8]
+        pythonVersion: [3.6, 3.7, 3.8]
```

Comment:
I only had 3.6 and 3.8 because it seemed like testing every version is too much on every gated build (following this logic, we should test on macos and windows as well on every gated build, but then it becomes way too slow since the windows builds are much slower), we have nightly builds anyway that test every combination.  But, if you think that it is still very important, I suppose we can add the extra test run.

Reply:
Any reason we should have the nightly builds? Given that we have so may differences with respect to python versions. we should test combinations even if it is slow. The nightly will fail anyways if we check in any code that has issues with 3.7.

Reply:
the nightly builds run all of the possible combinations.  the windows PR gates are really slow, it takes much longer than linux for some reason.  for PRs, we didn't think it was necessary to run every possible combination.  the nightly build exhaustively tests everything.  probably at some point we will also need to have p3 vs p1 builds, where p3 contain all of the really long-running tests.

Reply:
Looking at the workflow runs under action, it seems like the CI-python build took 30 something minutes to run. Individual jobs run in parallel. So not sure how adding another python version is adding to the gate run time. I am not adding a windows build in this PR.

The nightly builds sounds like a great idea to test out more versions on different OSes. But I still feel we should test each python version on at least one OS in the PR gates.

Reply:
ok, maybe it's fine if we just test one more python version then